### PR TITLE
Documentation Updates for Changes Pane to Include File Change Display

### PR DESCRIPTION
**Pull request by pradeep-patil-mdx**

### CODE REVIEW

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md [hunk 1]
```diff
@@ -117,7 +131,7 @@ Studio Pro also attaches some information automatically:
 
 If you also changed Java source code, added widgets, or made other changes that affect files other than the app file, you will see a **Changes on disk** tab that shows you what disk changes you are about to commit. **Open containing folder** opens the folder with the file on disk. For files with the **Modified** status, you can use **Compare with original** that opens an external tool to show the differences.
```
Comment (tknippenberg) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md:132]: > **Open containing folder** opens the folder with the file on disk. For files with the **Modified** status, you can use **Compare with original** that opens an external tool to show the differences.
from my previous PR seems incorrect. Can you remove that, and describe what double-clicking the file in this dialog does?

**Comment by MariaShaposhnikova** [comment]
@pradeep-patil-mdx , thanks for the PR! Could you please sign CLA? (You only need to sign it once).
Is this PR for a particular release?

**Comment by pradeep-patil-mdx** [comment]
> @pradeep-patil-mdx , thanks for the PR! Could you please sign CLA? (You only need to sign it once). Is this PR for a particular release?

This is for 10.18. Thanks

**Comment by MariaShaposhnikova** [comment]
@pradeep-patil-mdx and @tknippenberg , does the changes-pane document (https://docs.mendix.com/refguide/changes-pane/) also need an update?

**Comment by tknippenberg** [comment]
> @pradeep-patil-mdx and @tknippenberg , does the changes-pane document (https://docs.mendix.com/refguide/changes-pane/) also need an update?

We decided not to change that now, and might address it in a follow-up PR.

**Review by MariaShaposhnikova — APPROVED** [review]

**Pull request merged**